app/util/data_preparation/jira_prepare_data.py

```diff
@@ -24,6 +24,9 @@
 
 ENGLISH = 'en_US'
 
+URL = JIRA_SETTINGS.server_url
+CLIENT = JiraRestClient(URL, JIRA_SETTINGS.admin_login, JIRA_SETTINGS.admin_password)
```

Comment:
Why do you decide to put `CLIENT` outside the `main` function? 
If you do not use `__check_rte_status` function in prepare_data.py, why do you place it here instead if jira_clients?
In this case you can import `client` and `__check_rte_status` from jira_clients?

Reply:
We can't import CLIENT from main().
Yes, I'll try to do it from jira_client